src/commcare_cloud/ansible/roles/postgresql_base/tasks/set_up_dbs.yml

```diff
@@ -78,22 +78,17 @@
   assert: { that: "item.host in groups.postgresql or item.host in groups.pg_standby or item.host == '127.0.0.1' or item.host in groups.citusdb_cordinator " }
   with_items: "{{ postgresql_dbs.all }}"
 
-- name: Create PostgreSQL databases
+- name: Update PostgreSQL database config
   become_user: "{{ db_is_remote and 'ansible' or 'postgres' }}"
   vars:
     ansible_ssh_pipelining: true
-  postgresql_db:
-    name: "{{ item.name }}"
-    state: present
+  postgresql_set:
```

Comment:
Does this still create the dbs?

Reply:
hmm, not sure how that happened. Updated